package.json

```diff
@@ -177,10 +177,10 @@
         "configuration": {
             "title": "HubSpot",
             "properties": {
-                "hubspot.beta": {
+                "hubspot.hublLinting": {
```

Comment:
We _could_ keep this, but it would do nothing

Reply:
I like the idea of keeping it so that the pattern is easy to follow for the next beta feature that we'd like to introduce